### PR TITLE
[13.0][ADD] account_invoice_check_picking_date: Add new module

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT31195

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
LGTM :+1: Tested locally

File: account_invoice_check_picking_date/wizards/invoice_stock_picking_date_wiz.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class PartnerRiskExceededWiz(models.TransientModel):
```
Comment (pedrobaeza) [inline account_invoice_check_picking_date/wizards/invoice_stock_picking_date_wiz.py]: Change this class name

Comment (carlosdauden) [reply account_invoice_check_picking_date/wizards/invoice_stock_picking_date_wiz.py]: Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**